owncloudApp/src/main/java/com/owncloud/android/ui/adapter/ReceiveExternalFilesAdapter.java

```diff
@@ -51,25 +51,22 @@
 public class ReceiveExternalFilesAdapter extends BaseAdapter implements ListAdapter {
 
     private Vector<OCFile> mImmutableFilesList;
-    private Vector<OCFile> mFiles;
+    private Vector<OCFile> mFiles = new Vector<>();
     private Context mContext;
     private Account mAccount;
     private FileDataStorageManager mStorageManager;
     private LayoutInflater mInflater;
     private OnSearchQueryUpdateListener mOnSearchQueryUpdateListener;
 
     public ReceiveExternalFilesAdapter(Context context,
-                                       Vector<OCFile> files,
                                        FileDataStorageManager storageManager,
-                                       Account account
-    ) {
-        mFiles = files;
+                                       Account account) {
         mImmutableFilesList = (Vector<OCFile>) mFiles.clone();
```

Comment:
`mImmutableFilesList` will be always empty since `mFiles` is always empty at this point, isn't it?